### DISCUSSION TITLE
TSOL-M800 G2 -- Inverter nicht erreichbar

### DISCUSSION THREAD

**Discussion by Loewe1991** [tbnobody/OpenDTU#1318] [Q&A]
Falsches Forum, kann gelöscht werden

**Comment by tbnobody** [comment]
Das hier ist das OpenDTU Projekt. Du verwendest AhoyDTU. Du solltest daher deine Frage im korrekten Forum stellen.

**Reply by Loewe1991** [reply]
Huch, wie konnte denn das passieren ? 
Sorry ! 

Thread kann gerne gelöscht werden.
